src/actions/projects.js

```diff
@@ -119,16 +118,27 @@ const getAvatarsForProjects = projects => {
 const getWorkflowsForProjects = projects => {
     const projectIds = projects.map((project) => project.id)
 
-    return apiClient.type('workflows').get({mobile_friendly: true, active: true, project_id: projectIds})
-        .then(workflows => {
-            workflows.forEach(workflow => {
-                workflow.mobile_verified = workflow.mobile_friendly && isValidMobileWorkflow(workflow)
-                const project = projects.find(project => project.id === workflow.links.project)
-                if (!project.workflows.find((projectWorkflow) => projectWorkflow.id === workflow.id)) {
-                    project.workflows = R.append(workflow, project.workflows)
+    let fetchPaginatedWorkflows = (params, _page = 1) => {
+        return apiClient.type('workflows')
+            .get({...params, ...{page: _page}})
+            .then((workflows) => {
+                workflows.forEach(workflow => {
+                    workflow.mobile_verified = workflow.mobile_friendly && isValidMobileWorkflow(workflow)
+
+                    const project = projects.find(project => project.id === workflow.links.project)
+                    if (!project.workflows.find((projectWorkflow) => projectWorkflow.id === workflow.id)) {
+                        project.workflows = R.append(workflow, project.workflows)
+                    }
+                })
+
+                if (workflows.length === 20) {
+                    var currentPage = _page + 1
```

Comment:
I think this should be `newPage`, but this is not at all blocking. However, if you're changing the iOS version string, you might want to change this as well.